airbyte-integrations/connectors/destination-bigquery-denormalized/src/main/resources/spec.json

```diff
@@ -118,6 +118,13 @@
                 "type": "string",
                 "examples": ["data_sync/test"]
               },
+              "part_size_mb": {
+                "title": "Block Size (MB) for GCS multipart upload",
+                "description": "This is the size of a \"Part\" being buffered in memory. It limits the memory usage when writing. Larger values will allow to upload a bigger files and improve the speed, but consumes9 more memory. Allowed values: min=5MB, max=525MB Default: 5MB.",
+                "type": "integer",
+                "default": 5,
+                "examples": [5]
+              },
```

Comment:
[Optional] Please take a look if we have an option to limit the input. To make possible enter only integers in range  5 - 525.